_gsocproposals/2020/proposal_CMS-Monitoring.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Intelligent Alert Management System For HEP experiments
+layout: gsoc_proposal
+project: CMS-Monitoring
+year: 2020
+organization: CERN
+---
+
+## Description
+
+The growth of distributed services introduces a challenge to properly
+monitor their status and reduce operational costs. In CMS experiment
+at CERN we deal with distributed computing infrastructure which includes
+central services for authentication, workload management, data management,
+databases, etc. To properly operate and maintain this infrastructure
+we rely on various open-source monitoring tools, including 
+ElasticSearch, Kafka, Grafana stack (used by central CERN MONIT infrastructure), 
+Prometheus, AlertManager, VictoriaMetrics software components (used by the experiment),
+as well as custom solutions like GGUS WLCG or ServiceNow ticketing systems.
+
+On daily basis these CMS computing infrastructure may produce significant
+amount of information about various anomalies, interminent problems, outages as
+well as undergo scheduled maintance. Therefore the amount of alert
+notifications and tickets our operational teams should handle is very large.
+We're working towards Operataional Intellignet System which aiming to detect,
```

Comment:
also: "towards Operataional Intellignet System" -> change into "towards, an Operational Intelligent System". Using a spell checker for the entire text is recommended since I might have missed few typos.